### ISSUE TITLE
Add backend information to info page

### ISSUE THREAD

**Issue by dcolligan** [ga4gh/server#245]
It would be nice to have some information about the server configuration on the html page.

**Comment by jeromekelleher** [comment]
Also, it would be good to give some abstract information on the data, so user's can see what they are serving at a glance. For example, we might show a table of the variant sets and reads, like
```
variants
    1kgPhase1           123554346 
    1kgPhase3           123554346789
reads
    someReadSet      123432
```
This should be fairly simple to do (VariantSet should implement a function called getNumVariants(), for example), and would be an important usability bonus.

**Comment by jeromekelleher** [comment]
Working on this.

**Comment by pgrosu** [comment]
+1 that would be a big help.  I had to use mostly experimenting with the code - and some of the posts - to guide me with the clarification on how data worked its way through the code and how some of the functionalities worked together.

**Comment by dcolligan** [comment]
Fixed in #248
